soccer/gameplay/plays/offense/free_goal.py

```diff
@@ -0,0 +1,50 @@
+import play
+import behavior
+import robocup
+import skills.line_kick
+import tactics.defense
+import main
+import constants
+import enum
+import role_assignment
+
+
+class FreeGoal(play.Play):
```

Comment:
I guess we could make a playbook for when there are no functioning robots on the opposite team. This particular play should probably be put in with the 'real' plays since its our most basic offense if we are short on robots. Its pretty much Basic122 without the distraction robots.